third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-export-run-expected.txt

```diff
@@ -2,11 +2,11 @@ Tests that exporting works.
 
 ++++++++ testExportHtml
 
-# of .lh-audit divs (original): 137
+# of .lh-audit divs (original): 136
```

Comment:
What changed here?

Reply:
file last changed with csp-xss https://github.com/GoogleChrome/lighthouse/commits/master/third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-export-run-expected.txt

I don't see why this would have changed, since the audit is in experimental, so doesn't show in DT.